news/fix_dircolors.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Prevent exception on start up if ``dircolors`` util is wrong.
```

Comment:
It should show a error message in case of failure to read from dircolors. Otherwise the user should be given option to not to use dircolors at all.

Reply:
I think if there is an issue in the binary it should be shown to the user atleast. If this is a edge case, we can keep the current logic .